package.json

```diff
@@ -59,7 +59,9 @@
     "@material-ui/icons": "5.0.0-alpha.22",
     "@material-ui/lab": "5.0.0-alpha.22",
     "@msgpack/msgpack": "^2.3.0",
+    "@nebulous/skynet": "^2.0.1",
```

Comment:
hi @developerfred  I'm sorry but you need to rebase this PR. We've switched to pnpm last week and those dependencies now should be added in the `packages/maskbook/package.json` now.